cg/server/api.py

```diff
@@ -106,14 +106,14 @@ def families():
         count = len(records)
     else:
         customer_obj = None if g.current_user.is_admin else g.current_user.customer
-        families_q = db.families(
+        case_q = db.families(
```

Comment:
what do you suggest?

Reply:
I suggest that the name reveals as much as possible about what it is. case_q does not say much. I think you are better suited to name it since you know what is happening

Reply:
renamed to case_query